mesonbuild/interpreterbase/interpreterbase.py

```diff
@@ -59,11 +59,9 @@
         T.Callable[[mparser.BaseNode, T.List[TYPE_var], T.Dict[str, TYPE_var]], TYPE_var]
     ]
 
-
 class InvalidCodeOnVoid(InvalidCode):
-
-    def __init__(self, op_type: str) -> None:
-        super().__init__(f'Cannot perform {op_type!r} operation on void statement.')
+    def __init__(self, op_type: str, **kwargs: T.Any) -> None:
```

Comment:
I did that originally, but it wasn't supported back then:
https://github.com/mesonbuild/meson/pull/12154#issuecomment-1752164147

Great if it's working now!

Reply:
It's working for me with mypy 1.9.0, not sure if it works with 1.8.0. This may put is in an interesting place with Python 3.7 support, since mypy 1.9 doesn't support Python 3.7.

I'd like to use this pattern in the mlog module as well, which has a similar set of wrapper functionality.